scripts/check-imports

```diff
@@ -1,22 +1,27 @@
 #!/bin/sh
 
-# Generated type definitions should not include default imports.
-
 BASE_DIR="$(dirname "$0")/..";
 
-OUTPUT_FILE="/dist/react-stripe.d.ts"
+# Generated type definitions should not include default imports.
```

Comment:
(0) I think we might need to `set -e` in this file. Otherwise if the first `checkImport` call fails but the second succeeds, the script will exit with a 0 exit code. FWIW I put plop [`set -euo pipefail`](http://redsymbol.net/articles/unofficial-bash-strict-mode/) at the top of any script I write.

Reply:
Oh whoops, saw that both calls are joined with `&&`. Nvm!